roles/httpd/tasks/main.yml

```diff
@@ -1,22 +1,9 @@
 ---
-- name: Install Apache httpd
-  ansible.builtin.package:
-    name:
-      - httpd
-      - mod_ssl
+- name: Pull Apache container image
+  containers.podman.podman_image:
+    name: httpd:latest
```

Comment:
What image is this? I'd prefer we use fully qualified image names to know where from and what image we are targeting. Is this from docker hub? Quay?

Currently our containers are all CentOS 9 Stream based and I think that's a good standardization that we should aim for rather than mixing and matching.

Reply:
That was just a draft i wrote, I just realized, the code was not updated(though things are still breaking so it is WIP) But I'm using this ` name: registry.access.redhat.com/ubi9/httpd-24:latest` https://catalog.redhat.com/software/containers/ubi9/httpd-24/61a60c3e3e9240fca360f74a